src/view/utils/fetchFromEdge.js

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import getBaseRequestHeaders from "./getBaseRequestHeaders";
+import UserReportableError from "../errors/userReportableError";
+import * as HTTP_STATUS from "../constants/httpStatus";
+import * as NETWORK_ERROR_MESSAGE from "../constants/networkErrorMessage";
+import getHost from "./getHost";
+
+const EDGE_HOST_PROD = "https://edge.adobe.io";
+const EDGE_HOST_STAGING = "https://edge-stage.adobe.io";
+
+const ERROR_TYPE_INVALID_ACCESS_TOKEN =
+  "https://ns.adobe.com/aep/errors/EXEG-3032-401";
+
+export default async ({ orgId, imsAccess, path, params, headers, signal }) => {
+  const host = getHost({
+    imsAccess,
+    productionHost: EDGE_HOST_PROD,
+    stagingHost: EDGE_HOST_STAGING
+  });
```

Comment:
Previously we were just using the prod endpoint for edge configurations, but I figured we should be consistent with what we're doing for the aep schema endpoints and support both staging and production.